cli/azd/pkg/alpha/alpha_feature_manager.go

```diff
@@ -84,6 +85,11 @@ func (m *FeatureManager) IsEnabled(featureId FeatureId) bool {
 		return true
 	}
 
+	// For testing, allow enabling alpha features via the environment.
```

Comment:
@ellismg Would you have concerns if we made this `AZD_ENABLE_ALPHA_<feature>`, and applied `strconv.ParseBool`? Seems useful for CI scenarios

Reply:
No, that works for me, I will make the change.